lib/bloc/backup/backup_bloc.dart

```diff
@@ -389,6 +387,7 @@ class BackupBloc {
     try {
       await _saveLightningFees();
       await _savePosDB();
+      await _savePodcastsDB();
```

Comment:
wasn't the podcast functionality removed from the wallet?

Reply:
Oh ... yes, thx